api/tests/test_block_processor.py

```diff
@@ -140,8 +135,8 @@ async def main():
         retrievedTokenTransfers=[
             RetrievedTokenTransfer(transactionHash='0x24de708a8a36d16a68c69d973b3d2126c0d9fd01669335839697f71f7d50d397', registryAddress='0x60E4d786628Fea6478F785A6d7e704777c86a7c6', tokenId='27160', fromAddress='0xC310e760778ECBca4C65B6C559874757A4c4Ece0', toAddress='0x67D5fa7b939899d088A9d548EB9490a379deB912', operatorAddress='0x67D5fa7b939899d088A9d548EB9490a379deB912', amount=1, value=16005000000000000000, gasLimit=314274, gasPrice=82944034756, blockNumber=13898509, tokenType='erc721'),
             RetrievedTokenTransfer(transactionHash='0x4276409534cc165ef7872a336ed8db65c40428de163406f2b3cf9c4ecdfa9d7b', registryAddress='0x828B78e11BC6A061b8b1AB9CF29C68ED6Abb247B', tokenId='354', fromAddress='0x59CacaD5f405C369E092be30c7B571384fcB0A9d', toAddress='0xa511Bf40cF8A5090a3cc19C6DF38dbCC1DC755A3', operatorAddress='0xa511Bf40cF8A5090a3cc19C6DF38dbCC1DC755A3', amount=1, value=90000000000000000, gasLimit=313653, gasPrice=79444034756, blockNumber=13898509, tokenType='erc721'),
-            RetrievedTokenTransfer(transactionHash='0x6bb37dcb9a60ed49da56ccfb21dfcae886e81debd3ab94d73497d62744e758d2', registryAddress='0x57f1887a8BF19b14fC0dF6Fd9B2acc9Af147eA85', tokenId='53091478429947851570161017064535674537231673781900674689421707367586767084753', fromAddress='0x0000000000000000000000000000000000000000', toAddress='0x283Af0B28c62C092C9727F1Ee09c02CA627EB7F5', operatorAddress='0x04c1f61baF8DE8DA75DB52f09D9d92a1592f46eA', amount=1, value=1446765707746493, gasLimit=295604, gasPrice=79444034756, blockNumber=13898509, tokenType='erc721'),
-            RetrievedTokenTransfer(transactionHash='0x6bb37dcb9a60ed49da56ccfb21dfcae886e81debd3ab94d73497d62744e758d2', registryAddress='0x57f1887a8BF19b14fC0dF6Fd9B2acc9Af147eA85', tokenId='53091478429947851570161017064535674537231673781900674689421707367586767084753', fromAddress='0x283Af0B28c62C092C9727F1Ee09c02CA627EB7F5', toAddress='0x04c1f61baF8DE8DA75DB52f09D9d92a1592f46eA', operatorAddress='0x04c1f61baF8DE8DA75DB52f09D9d92a1592f46eA', amount=1, value=1446765707746493, gasLimit=295604, gasPrice=79444034756, blockNumber=13898509, tokenType='erc721'),
+            RetrievedTokenTransfer(transactionHash='0x6bb37dcb9a60ed49da56ccfb21dfcae886e81debd3ab94d73497d62744e758d2', registryAddress='0x57f1887a8BF19b14fC0dF6Fd9B2acc9Af147eA85', tokenId='53091478429947851570161017064535674537231673781900674689421707367586767084753', fromAddress='0x0000000000000000000000000000000000000000', toAddress='0x283Af0B28c62C092C9727F1Ee09c02CA627EB7F5', operatorAddress='0x04c1f61baF8DE8DA75DB52f09D9d92a1592f46eA', amount=1, value=723382853873246, gasLimit=295604, gasPrice=79444034756, blockNumber=13898509, tokenType='erc721'),
```

Comment:
this test case shows another example we need to be looking out for - its the same token being transferred so i dont think we should divide here. in fact im actually not sure what to do here at all.

Reply:
https://etherscan.io/tx/0x6bb37dcb9a60ed49da56ccfb21dfcae886e81debd3ab94d73497d62744e758d2